File: language/benchmarks/tokens/cases/topology-http-test-world/topology-http-test-world.ts
import { mailerApi } from "./topology";
import { systemClock } from "./world/clock";
import { realFs } from "./world/fs";
import { proxy } from "./world/http";
import { capture } from "./world/log";
import { deny } from "./world/process";
import { buildWorld } from "./world/runtime";
import { virtual } from "./world/timer";

export const world=buildWorld(systemClock(),realFs(),[proxy("http://127.0.0.1:45110",mailerApi)],capture(),deny(),[],virtual());

File: language/benchmarks/tokens/cases/topology-http-test-world/topology-http-test-world.py
from topology import mailer_api
from world.clock import system_clock
from world.fs import real
from world.http import proxy
from world.log import capture
from world.process import deny
from world.runtime import build_world
from world.timer import virtual


world = build_world(system_clock(), real(), [proxy("http://127.0.0.1:45110", mailer_api)], capture(), deny(), [], virtual())
